news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md

```diff
@@ -0,0 +1,98 @@
+
+With 76 total entries (resulting in 33 submitted entries) the competition was fierce, and the judges had a lot of work ahead of them. After much deliberation, the dust has settled, and we are proud to present the top 3 entries.
+
+## Winners
+
+### 3rd place - Team RGB
+
+Drum and bass enjoyers can rejoice as Team RGB harnessed the full power of neurofunk. Featuring music by [Artackni](https://artackni.newgrounds.com/) and craftsmanship of [Amon-](https://osu.ppy.sh/users/18161041) and ballsmaster69, [*Encaged*](https://osu.ppy.sh/beatmapsets/2242804#osu/4767104) is a slider-heavy tech map that is bound to get the adrenaline pumping and keep your attention throughout its course.
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://assets.ppy.sh/media/news/kyoku-2024-3rd-place.mp4" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Congratulations to Team RGB as they scored themselves 3rd place with a final score of **84.50**!
+
+### 2nd place - Zero Blunders
+
+Team Zero Blunders has put their best foot forward in this contest with a genre mash-up song featuring many osu! references. [*who's afraid?*](https://osu.ppy.sh/beatmapsets/2242420#osu/4766280) by fur:trash (also known as [Nihkee](https://osu.ppy.sh/beatmapsets/2242420#osu/4766280)) offers many chances for [Mysty](https://osu.ppy.sh/users/10210657), [Le Mirai](https://osu.ppy.sh/users/13646997), [hakashii](https://osu.ppy.sh/users/11688893) & [[-Evil-]](https://osu.ppy.sh/users/10234313) to express their mapping ideas through music genres such as dariacore, dubstep & hyperpop, creating a balanced map that is both fun to look at and play.
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://assets.ppy.sh/media/news/kyoku-2024-2nd-place.mp4" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Congratulations to Zero Blunders for managing to lock in a total score of **86.00**, securing 2nd place and 1 contest point for all team members.
+
+### 1st place - cosmo/LOGIΛ
+
+Team cosmo/LOGIΛ have managed to concoct what can best be described as a Grand Finals Tiebreaker map on steroids. [Stellar Evolution](https://osu.ppy.sh/beatmapsets/2242303) from [airlemoneX](https://soundcloud.com/airlemonex) has hints of [Camellia](https://osu.ppy.sh/beatmaps/artists/31) as the inspiration behind the track, with expert mappers [Lulu-](https://osu.ppy.sh/users/4201715), [Slifer](https://osu.ppy.sh/users/15084122) and [Halgoh](https://osu.ppy.sh/users/4109923) fully playing into it with a focus on visuals that do not compromise gameplay.
```

Comment:
```suggestion
Team cosmo/LOGIΛ managed to concoct what can best be described as a Grand Finals tiebreaker map on steroids. [Stellar Evolution](https://osu.ppy.sh/beatmapsets/2242303) from [airlemoneX](https://soundcloud.com/airlemonex) has hints of [Camellia](https://osu.ppy.sh/beatmaps/artists/31) as the inspiration behind the track, with expert mappers [Lulu-](https://osu.ppy.sh/users/4201715), [Slifer](https://osu.ppy.sh/users/15084122) and [Halgoh](https://osu.ppy.sh/users/4109923) fully playing into it with a focus on visuals that do not compromise gameplay.
```

Reply:
> visuals that do not compromise gameplay

hmm sure but it's quite hard to play as a result of those visuals sometimes O.o

Reply:
https://github.com/ppy/osu-wiki/pull/12165/commits/23d01913b31d87fb86f160830ab3fd52a73fff98